zyte_common_items/pipelines.py

```diff
@@ -53,3 +57,49 @@ def __init__(self):
 
     def process_item(self, item, spider):
         return ae.downgrade(item)
+
+
+class DropItem(Exception):
+    """Drop item from the item pipeline."""
+
+    pass
```

Comment:
I think this class need to be `scrapy.exceptions.DropItem`, otherwise the items would not actually be dropped and this custom exception would be raised instead.

Reply:
Agree, wanted to avoid import scrapy, but we imported this anyway for utils